setenv.sh

```diff
@@ -11,26 +11,28 @@ else
 fi
 
 export TORQHOME=${dirpath}
+export TORQAPPHOME=${TORQHOME}
+export TORQDATAHOME=${TORQHOME}
 export KDBCONFIG=${TORQHOME}/config
 export KDBCODE=${TORQHOME}/code
-export KDBLOG=${TORQHOME}/logs
+export KDBLOG=${TORQDATAHOME}/logs
 export KDBHTML=${TORQHOME}/html
 export KDBLIB=${TORQHOME}/lib
-export KDBHDB=${TORQHOME}/hdb
-export KDBWDB=${TORQHOME}/wdbhdb
-export KDBDQCDB=${TORQHOME}/dqe/dqcdb/database
-export KDBDQEDB=${TORQHOME}/dqe/dqedb/database
-export KDBTPLOG=${TORQHOME}/tplogs
+export KDBHDB=${TORQDATAHOME}/hdb
+export KDBWDB=${TORQDATAHOME}/wdbhdb
+export KDBDQCDB=${TORQAPPHOME}/dqe/dqcdb/database
+export KDBDQEDB=${TORQAPPHOME}/dqe/dqedb/database
```

Comment:
These should be `TORQDATAHOME` too I think, they're just more databases.

Reply:
Yeah that is fair shout. Just need to move it to correct place in the install script im doing as well. But makes sense. Will make that change.